docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+  webpack:
+    container_name: dim-webpack
+    build:
+      context: ./docker-compose/webpack
+    command: bash -c "yarn install && yarn start"
+    restart: always
+    volumes:
+      - .:/usr/src/app
```

Comment:
See comment below about using named volume vs host-mounted filesystem.

Reply:
Good feedback, I'll work on it